public/app/features/canvas/runtime/element.tsx

```diff
@@ -385,7 +392,7 @@ export class ElementState implements LayerElement {
   render() {
     const { item } = this;
     return (
-      <div key={`${this.UID}`} style={{ ...this.sizeStyle, ...this.dataStyle }} ref={this.initElement}>
+      <div key={this.UID} ref={this.initElement}>
```

Comment:
👍 -- this means we are applying all styles directly via javascript in `updateLayout()` now.

Let make that very clear in the comments/javadoc for the function.  Perhaps change the name 🤔